_data/changelogs/component-time-picker-accessibility.yml

```diff
@@ -0,0 +1,8 @@
+title: Time picker accessibility tests
+type: component
+items:
+ - date: NNNN-NN-NN
+   summary: Added accessibility tests page.
+   affectsGuidance: true
+   githubPr: [Related PR number]
```

Comment:
Just letting you know the html errors are just because this is expecting the PR number here! Adding `2921` should fix the build errors you're seeing

Reply:
Thank you for pointing that out! Resolved in [f9a36fa](https://github.com/uswds/uswds-site/pull/2921/commits/f9a36fac82d4f4f921c213f53d371a43a02ac343)